### DISCUSSION TITLE
Complete Dynamics integration for did codes (customer/project)

### DISCUSSION THREAD

**Discussion by pettersh** [Puzzlepart/did#1093] [Feature request]
We should prioritize the completion of this integration @okms

**Comment by olemp** [comment]
What is missing to complete this, @pettersh?

**Reply by pettersh** [reply]
I have no idea. I know I have to create did codes in both Dynamics and Did, and would like to only create them once in the future. I don't know what is the blocker, if it's technical or time/availability constraints.
